### ISSUE TITLE
Update 'Writing Transactions to a Ledger for an Un-privileged Author.md' to use Aries-VCR

### ISSUE THREAD

**Issue by WadeBarnes** [bcgov/von-network#135]
Update the instructions in Writing Transactions to a Ledger for an Un-privileged Author to use [aries-vcr](https://github.com/bcgov/aries-vcr).

The instructions currently use [TheOrgBook](https://github.com/bcgov/TheOrgBook) which has been replaced by [aries-vcr](https://github.com/bcgov/aries-vcr) and retired.

The same basic processes apply to [aries-vcr](https://github.com/bcgov/aries-vcr), and [aries-vcr-issuer-controller](https://github.com/bcgov/aries-vcr-issuer-controller).

**Comment by EtricKombat** [comment]
Hi @WadeBarnes , a doubt regarding the repo : https://github.com/bcgov/aries-vcr

Does the name "Hyperledger Indy Catalyst" need to replaced by "aries-vcr" , i'm just trying to understand this.

**Comment by swcurran** [comment]
Yes.  We are far behind on updating the documentation for the repo.  Hyperledger Indy Catalyst was the working title, but it had to change.

**Comment by EtricKombat** [comment]
Ok, thank you @swcurran  for confirming that.
